tests/Doctrine/Tests/ORM/Functional/SQLFilterTest.php

```diff
@@ -401,7 +431,7 @@ public function testQueryGenerationDependsOnFilters(): void
         $conf = $this->_em->getConfiguration();
         $conf->addFilter('country', '\Doctrine\Tests\ORM\Functional\CMSCountryFilter');
         $this->_em->getFilters()->enable('country')
-            ->setParameter('country', 'en', DBALType::STRING);
+            ->setParameter('country', ['en'], DBALType::STRING);
```

Comment:
Why changing an existing test and filter?

Reply:
Because we have two filters in the test, one for locale one for country. By changing one to be a list we maximize test coverage without adding more tests.